typings/codeceptjs.d.ts

```diff
@@ -290,9 +290,10 @@ declare namespace CodeceptJS {
     seeEventSubmissionConfirmation(event: string) : void,
     clickHyperlink(link: string, urlNavigatedTo: string) : void,
     startEventViaHyperlink(link: string) : void,
-    seeDocument(title: string, name: string, status?: string, reason?: string) : void,
+    seeDocument(title: string, name: string, status?: string, reason?: string, dateAndTimeUploaded?: string, uploadedBy?: string) : void,
```

Comment:
I might be blind but I don't see the corresponding change in the `steps_file.js`

Reply:
its there :)

Reply:
fixed it now.